lms/djangoapps/discussion_api/forms.py

```diff
@@ -0,0 +1,33 @@
+"""
+Discussion API forms
+"""
+from django.core.exceptions import ValidationError
+from django.forms import Form, CharField, IntegerField
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.locator import CourseLocator
+
+
+class ThreadListGetForm(Form):
+    """
+    A form to validate query parameters in the thread list retrieval endpoint
```

Comment:
@andy-armstrong We should use this approach for validating query parameters for the Teams API.